leshan-client-demo/src/main/java/org/eclipse/leshan/client/demo/LeshanClientDemo.java

```diff
@@ -456,6 +469,47 @@ public static void main(final String[] args) {
             }
         }
 
+        List<Certificate> trustStore = null;
+        if (cl.hasOption("truststore")) {
+            trustStore = new ArrayList<>();
+
+            String trustStoreName = cl.getOptionValue("truststore");
+
+            if (trustStoreName.startsWith("file://")) {
+                // Treat argument as Java trust store
+                try {
+                    Certificate[] trustedCertificates = SslContextUtil.loadTrustedCertificates(trustStoreName);
+                    trustStore.addAll(Arrays.asList(trustedCertificates));
+                } catch (Exception e) {
+                    System.err.println("Unable to load trust store : " + e.getMessage());
+                    e.printStackTrace();
+                    formatter.printHelp(USAGE, options);
+                }
+            } else {
+                // Treat argument as file or directory
+                File input = new File(cl.getOptionValue("truststore"));
+
+                // check input exists
+                if (!input.exists())
+                    throw new FileNotFoundException(input.toString());
```

Comment:
I know this is like this in leshan-server-demo but it sounds not so consistent.

All errors seems to rather be handled by a : 
```
System.err.println("Error message" );
formatter.printHelp(USAGE, options);
return;
```

Reply:
In my opinion quite common and good approach is to give clear error message if something is wrong in command line arguments and then exit with error (System.exit(<non-zero>).

What I tried to follow here was what was code in there.

I can change it how you want it to be. I would like to keep them in sync in all three demos.

Perhaps something like:
```java
                System.err.println("Failed to load trust store: file or directory does not exist: " + input.toString());
                // or LOG.error() ?
                //?? formatter.printHelp(USAGE, options);
                return;
```

Shall I change also:
```java
                System.err.println("Failed to load trust store: " + e.getMessage());
                // or LOG.error() ?
                //?? formatter.printHelp(USAGE, options);
                return;
```
(and drop stack printing)

LOG facility has throwable argument support too. So it might provide common formatting for exceptions if this is something you wish to have.

Reply:
> I can change it how you want it to be. I would like to keep them in sync in all three demos.

I agree.

> I can change it how you want it to be.

Thx :pray:

Reply:
Please check if the current messages are OK.